hudi-client/src/test/resources/org/apache/hudi/index/hbase/TestHBaseIndex.properties

```diff
@@ -0,0 +1,38 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+# HoodieWriteConfig
```

Comment:
whats the value of pull this into config instead of code? IMO, code is much easier to debug..

Reply:
i agree that it's handy with builder pattern. I came from the point where most configs across test cases are static and duplicated; for each test class, only 1 or 2 properties to variate. That's what the minimal exposure via overwritingProps aim for. But i agree the benefits are less obvious. closing this one first. thanks for the input.